src/core/agent/modules/signify/signifyApi.ts

```diff
@@ -474,15 +474,36 @@ export class SignifyApi {
     name: string;
   }> {
     const icp = exn.e.icp;
-    const rstates = exn.a.rstates;
+
```

Comment:
So we won't trust the `rstates` that were sent in the exchange anymore?

Reply:
As far as I can tell, `rstates` is not part of the inception message.